### DISCUSSION TITLE
How to update RAVEN toolbox to the latest version

### DISCUSSION THREAD

**Discussion by sysyboom** [SysBioChalmers/RAVEN#448] [Help me!] [answered]
Thank you for keeping the software updated！
I have installed the version 2.6.1 several months ago, and then how could I update my version to 2.7.6?

**Comment by edkerk** [comment, marked as answer]
Great that you're using RAVEN, and it is indeed recommended to use the latest version.

This spring/summer, we updated the information regarding installation and upgrading, available [here](https://github.com/SysBioChalmers/RAVEN/wiki/Installation#obtain-raven-toolbox). As you can see, the process of upgrading depends a bit on how you have installed RAVEN.

**Reply by sysyboom** [reply]
Thanks for your answer！
